hw/bsp/stm32f429discovery/src/hal_bsp.c

```diff
@@ -42,17 +42,18 @@
 #if MYNEWT_VAL(UART_0)
 static struct uart_dev hal_uart0;
 
+/* UART connected to st-link */
 static const struct stm32_uart_cfg uart_cfg[UART_CNT] = {
     [0] = {
-        .suc_uart = USART2,
-        .suc_rcc_reg = &RCC->APB1ENR,
-        .suc_rcc_dev = RCC_APB1ENR_USART2EN,
-        .suc_pin_tx = MCU_GPIO_PORTD(5),	/* PD5 */
-        .suc_pin_rx = MCU_GPIO_PORTD(6),	/* PD6 */
+        .suc_uart = USART1,
+        .suc_rcc_reg = &RCC->APB2ENR,
+        .suc_rcc_dev = RCC_APB2ENR_USART1EN,
+        .suc_pin_tx = MCU_GPIO_PORTA(9),
```

Comment:
You might want to send a note out when changing console UART. If there's someone using this BSP, they'll be surprised after pulling in this change.

Reply:
That is a good question. Having console go via that seems like a better option for ease of use.